scripts/detection/ssd/eval_ssd.py

```diff
@@ -159,6 +186,58 @@ def validate(net, val_data, ctx, classes, size, metric):
         val_dataset, args.data_shape, args.batch_size, args.num_workers)
     classes = val_dataset.classes  # class names
 
+    # calibration
+    if args.calibration and not args.quantized:
+        exclude_layers = []
+        exclude_layers += ['ssd0_concat0',
```

Comment:
anyway to avoid hardcoded exclude_layers? Will wildcard matching or regex supported by quantize_net? Further, will quantize_net handle the default operators that are not supported in quantization?

Reply:
good suggestions, will enhance it soon. BTW, there comes bugs in quantized_concat and quantized_flatten, will take a look at them. By default, quantize_net api will exclude all fc layers since quantized_fc's weight will be saved as int8 type, which cannot be load back as a gluon block now.